src/main/java/seedu/address/storage/JsonAdaptedPlan.java

```diff
@@ -0,0 +1,91 @@
+package seedu.address.storage;
+
+import static seedu.address.logic.Messages.MESSAGE_PERSON_DOES_NOT_EXIST;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.AddressBook;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+import seedu.address.model.plan.Plan;
+import seedu.address.model.plan.PlanDateTime;
+import seedu.address.model.plan.PlanName;
+
+
+
+
+/**
+ * Jackson-friendly version of {@link Plan}.
+ */
+class JsonAdaptedPlan {
```

Comment:
This class looks good! It is preferred to add tests for this class. Maybe you can add it next PR!